### DISCUSSION TITLE
Use PAIR-EXP for double/single quote character?

### DISCUSSION THREAD

**Discussion by ghost** [meow-edit/meow#320] [Q&A] [answered]
PAIR-EXP does not seems to be working with double/single quote character, ie
````elisp
(meow-thing-register 'qoute
                     '(pair ("\"") ("\""))
                     '(pair ("\"") ("\"")))
````
have no effects, while full width quote characters like `“ ”` does work.
I need this because the string thing comes with meow don't select double quotes pairs in comments, is it possible to done it without regex or syntax table?

**Comment by eshrh** [comment, marked as answer]
Not immediately certain why this is. It appears that the issue is with PAIR-EXPS where the start and end character is the same. Can you please file an issue so I remember to investigate?

To be honest, I'm not sure why PAIR-EXP exists, since it's afaik strictly inferior to the REGEXP-EXPR.  This should work fine for now
```elisp
(meow-thing-register 'quote
                     '(regexp "\"" "\"")
                     '(regexp "\"" "\""))
```

**Reply by ghost** [reply]
Ok I'll open a issue on this, will use the regex for now, but think PAIR-EXPR might be better because `meow--thing-pair-function` will check if the pair token are actually match.
